docs/content/en/tasks/backup-restore/_index.md

```diff
@@ -115,32 +115,27 @@ To perform a backup of a Cassandra datacenter, create the following custom resou
 
 ```yaml
 apiVersion: medusa.k8ssandra.io/v1alpha1
-kind: CassandraBackup
+kind: MedusaBackupJob
 metadata:
   name: medusa-backup1
 spec:
   cassandraDatacenter: dc1
-  name: medusa-backup1
 ```
 
-The `metadata.name` value can match the `spec.name` value for convenience, but it is not mandatory. The latter will be used to identify the backup in the storage backend, the former being the name of the `CassandraBackup` custom resource in Kubernetes.
-
```

Comment:
The **Creating a Backup** section says:

>To perform a backup of a Cassandra datacenter, create the following custom resource in the namespace where K8ssandra was deployed:

Can you please change this to make it clear it needs to be deployed in the same namespace and in the same k8s cluster in which the CassandraDatacenter exists.

Reply:
👍